ClinGen/clincoded
# Connects to #1374. Connects to #1370. Connects to #1371. Connects to #1376.

**Steps to test #1370:**
1. In GCI, create a new Experimental Data evidence.
2. Select **Biochemical Function** or **Functional Alteration** _Experiment type_.
3. If **Biochemical Function** was selected, select **option A** for the _Please select which one (A or B) you would like to curate_ option.
4. Expect to see "Search GO using the OLS." with the linkout instead of "Search for a GO term using the OLS...":
![image](https://user-images.githubusercontent.com/6956310/27161338-8d236d14-512f-11e7-833d-a45870cf2a64.png)

**Steps to test #1371:**
1. In GCI, create a new Experimental Data evidence.
2. Select **Biochemical Function** or **Functional Alteration** _Experiment type_.
3. Select **option A** for the _Please select which one (A or B) you would like to curate_ option.
4. Expect to see the _disabled_ **Shared disease** input textarea displaying the MonDO disease term and clique leader id instead of displaying **ORPHAUndefined**:
![image](https://user-images.githubusercontent.com/6956310/27161402-0d0f92dc-5130-11e7-980a-1c414bac7a27.png)

**Steps to test #1374:**
1. In GCI, create a new Proband Individual evidence.
2. Fill out the required fields. In the **Score Proband** section, select **Score** from the _Select Status_ dropdown. And then revert the selection back to **No Selection**.
3. Save the evidence and go to the view-only page of this evidence.
4. Expect to see _NO_ **Other Curator Scores** section being displayed, and the **Score Proband** section only displays the yellow warning note instead of the scoring form fields.

**Steps to test #1376:**
1. In GCI, create a new Non-Proband Individual evidence.
2. Fill out the required fields, but _DO NOT_ associate it with a disease.
3. Expect to be able to save the evidence successfully.
